src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyFactory.java

```diff
@@ -310,17 +353,8 @@ private MessageConsumer newCacheableRequestStream(
         long routeId,
         long initialId,
         long resolveId,
-        ArrayFW<HttpHeaderFW> headers,
-        int requestHash)
+        HttpProxyCacheableRequestGroup group)
     {
-        counters.requestsCacheable.getAsLong();
-        HttpProxyCacheableRequestGroup group = supplyCacheableRequestGroup(requestHash);
-
-        HttpHeaderFW authorizationHeader = headers.matchFirst(h -> AUTHORIZATION.equals(h.name().asString()));
-        if (authorizationHeader != null)
-        {
-            group.authorizationHeader(authorizationHeader.value().asString());
-        }
```

Comment:
With the adjustment to `defaultCache.matchCacheableRequest(...)` described above, this change can be reverted.